demo/resources.hpp

```diff
@@ -44,8 +44,8 @@ namespace resources
 
 	void CreateResources(wr::RenderSystem* render_system)
 	{
-		texture_pool = render_system->CreateTexturePool(256, 55);
-		material_pool = render_system->CreateMaterialPool(256);
+		texture_pool = render_system->CreateTexturePool(16*1024*1024, 55);
```

Comment:
Might be an idea to add _MB or _KB (same way as _deg); though we as a team might have to discuss that

Reply:
This. I won't except 1024*1024

Reply:
Done.